src/tests/unit/electron/adapters/electron-storage-adapter.test.ts

```diff
@@ -13,75 +13,50 @@ describe('ElectronStorageAdapter', () => {
         testKey1: 'test-value-1',
         testKey2: 'test-value-2',
     };
-    let electronStorageAdapter: ElectronStorageAdapter;
+
     let indexedDBInstanceMock: IMock<IndexedDBAPI>;
     let loggerMock: IMock<Logger>;
 
+    let testSubject: ElectronStorageAdapter;
+
     beforeEach(() => {
         indexedDBInstanceMock = Mock.ofType<IndexedDBAPI>();
         loggerMock = Mock.ofType<Logger>();
-        electronStorageAdapter = new ElectronStorageAdapter(indexedDBInstanceMock.object, loggerMock.object);
+        testSubject = new ElectronStorageAdapter(indexedDBInstanceMock.object, loggerMock.object);
     });
 
-    describe('setting user data with electron storage adapter works', () => {
-        it('sets user data with input items', async () => {
+    describe('setUserDataP', () => {
```

Comment:
Typo (extra P)

Reply:
Good catch... this came from when I have both callback-based and promise-base set user data functions. Will update